enterprise/api/v1/views.py

```diff
@@ -173,6 +179,11 @@ def basic_list(self, request, *arg, **kwargs):
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data)
 
+    @permission_required(
+        'enterprise.can_access_admin_dashboard', fn=lambda request, pk: pk)
```

Comment:
nit: does this need to split on 2 lines?

Reply:
nope 🥲